README.md

```diff
@@ -1 +1,105 @@
-# Stock-metrics
\ No newline at end of file
+![](https://img.shields.io/badge/stock_metrics-green)
+
+# Stock Metrics
+
+> Stock metrics is a web(mobile-focused) app for checking the companies listed on the stoc exchange and see their live metrics. Its data is consumed from the Financial modelling API. It is built using React, Redux, and Styled Components, Framer Motion. 
+> 
+
+<br/>
+
+# Presentation
+[Presentation Link](https://www.loom.com/share/826b3bcf225b438c9cc1651e917ce8e7)
+# Mini Demo 
+
+![](./src/images/demo.gif)
+
+# Live Demo
+
+[Live Link](https://stock-metrics.netlify.app/)
+
+<br/>
+
+Additional description about the project and its features.
+## Built With
+
+- Major languages (JavaScript)
+- Framework/Libraries (React, Styled-Components, Framer Motion, React Testing Library)
+- Technologies/tools used 
+  
+  ``` bash
+  - create-react-app tool
+  - Webpack(Code Bundlng et al)
+  - Babel (code transpiling)
+  - Jest(for testing)
+  - Git(version control)
+  - ESLint(JavaScript linting)
+  - Stylelint(style linting)
+
+  ```
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+ - A text editor(preferably Visual Studio Code)
+### Install
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+### Usage
+#### Clone this repository
+
+```bash
+$ git clone git@github.com:AnselemOdims/Stock-metrics.git
+$ cd Stock-metrics
+```
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+## Authors
+
+👤 **Anselem Odimegwu**
+
+- GitHub: [github](https://github.com/AnselemOdims)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/anselem-odimegwu/)
+
+<br>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AnselemOdims/Stock-metrics/issues).
+
+<br>
+
```

Comment:
- [ ] Please add the `Acknowledgements` section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the original design, as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Ok thanks